frontend/src/lib/components/ActivationSidebar/ActivationSidebar.tsx

```diff
@@ -0,0 +1,126 @@
+import { ProfessorHog } from '../hedgehogs'
+
+const Task = ({ id, name, description, completed, canSkip, skipped, url }: ActivationTaskType): JSX.Element => {
+    const displaySideAction = !completed && !skipped && canSkip
+    const { runTask, skipTask } = useActions(activationLogic)
+
+    const content = (
+        <div className="my-4 mx-2">
+            <p className="m-0">{name}</p>
+            {!completed && !skipped && <p className="font-normal text-xs mt-2 mb-0">{description}</p>}
+        </div>
+    )
+
+    const params: Partial<LemonButtonProps> = {
+        id,
+        fullWidth: true,
+        type: 'secondary',
+        icon: completed ? <IconCheckmark /> : skipped ? <IconClose /> : null,
+        status: completed ? 'primary-alt' : skipped ? 'muted' : undefined,
+        tooltip: name,
+    }
+    if (url) {
+        params.to = url
+        params.targetBlank = true
+    } else {
+        params.onClick = () => runTask(id)
+    }
+    return (
+        <li>
+            {displaySideAction ? (
+                <LemonButtonWithSideAction
+                    {...params}
+                    sideAction={{
+                        icon: <IconClose />,
+                        tooltip: 'Skip task',
+                        onClick: () => skipTask(id),
+                        status: 'muted',
+                    }}
+                >
+                    {content}
+                </LemonButtonWithSideAction>
+            ) : (
+                <LemonButton {...params}>{content}</LemonButton>
+            )}
+        </li>
+    )
+}
+
+const ActivationSidebar = (): JSX.Element => {
+    const { isActivationSideBarShown } = useValues(navigationLogic)
+    const { hideActivationSideBar } = useActions(navigationLogic)
+    const { activeTasks, completedTasks, completionPercent, showSessionRecordingConfig } = useValues(activationLogic)
+    const { setShowSessionRecordingConfig } = useActions(activationLogic)
+
+    return (
+        <div className={clsx('ActivationSideBar', !isActivationSideBarShown && 'ActivationSideBar--hidden')}>
+            <div className="ActivationSideBar__content pt-2 px-4 pb-16">
+                <div className="ActivationSideBar__close_button">
+                    <LemonButton
+                        icon={<IconClose />}
+                        onClick={() => {
+                            if (showSessionRecordingConfig) {
+                                setShowSessionRecordingConfig(false)
+                            } else {
+                                hideActivationSideBar()
+                            }
+                        }}
+                    />
+                </div>
+                {showSessionRecordingConfig ? (
+                    <SessionRecordingConfig />
+                ) : (
+                    <>
+                        <h2 className="subtitle">Quick Start</h2>
+                        <p>Learn how to get the most out of PostHog, straight away.</p>
```

Comment:
The `, straight away.` reads weird to me, and I (as a user) am not sure what it means to "get the most out of PostHog". I'd reword to something like...

```suggestion
                        <p>Use our Quick Start guide to learn about everything PostHog can do for you and your product.</p>
```